azurerm/resource_arm_storage_share_test.go

```diff
@@ -223,32 +297,107 @@ func testCheckAzureRMStorageShareDestroy(s *terraform.State) error {
 }
 
 func testAccAzureRMStorageShare_basic(rInt int, rString string, location string) string {
+	template := testAccAzureRMStorageShare_template(rInt, rString, location)
 	return fmt.Sprintf(`
-resource "azurerm_resource_group" "test" {
-  name     = "acctestRG-%[1]d"
-  location = "%[2]s"
+%s
+
+resource "azurerm_storage_share" "test" {
+  name                 = "testshare%s"
```

Comment:
Should this be
```suggestion
  name                 = "acctestShare%s"
```

Reply:
nope - this is a Data Plane API rather than a Resource Manager API; as such this won't be found directly by the Dalek. Instead the Dalek will find the Storage Account which uses the prefix `acctest` so this will get detected/deleted 👍